modules/alloy/src/main/ts/ephox/alloy/behaviour/positioning/PositionSchema.ts

```diff
@@ -9,7 +9,7 @@ const TransitionSchema = [
 ];
 
 export const PositionSchema = [
-  FieldSchema.defaulted('useFixed', Fun.never),
+  FieldSchema.defaulted('usePositioningType', Fun.constant('relative')),
```

Comment:
This type needs to be more restrictive. Boulder should fail if we pass through something random like "usePositioningType: random-guess". Look at what other functions there are in `FieldSchema`.

Reply:
(especially because of how the other part was implemented where if the positioningType wasn't one of a set of possible values, styles became undefined)

Reply:
- [ ] lock down possible values more through boulder.

Reply:
Came up with something like [this](https://github.com/tinymce/tinymce/pull/8294/commits/25526a655436472e32aaebbea9ed7dd3e9c9d8e6).